content/pages/privacy-policy.md

```diff
@@ -0,0 +1,105 @@
+title: Mixxx Privacy Policy
+slug: privacy-policy
+
+## Introduction
+
+Mixxx is a non-profit project, managed by volunteers.
+
+There is no business-model around Mixxx, and the only revenue we receive is through donations.
+
+For privacy policies and conformance to GDPR, a few distinctions will be done between:
+
+- Mixxx application
+- Main Mixxx website
+- Community Mixxx websites
+
+For each of the previous software/websites, we'll detail the different privacy policies.
+
+## Mixxx application
+
+The Mixxx application doesn't use any user account, and does not collect any user data, when running.
+
+Mixxx does not collect any data, nor any telemetry, when Mixxx is being run.
+However, some limited personal information could be collected or shared in 3 precise cases:
+
+- during the crash report process,
+- during updates checking,
+- for metadata retrieval.
+
+## Mixxx crash reporting
+
+The Mixxx application itself neither collects crash reports nor shares them.
+
+Crash reports are handled by the Stores/Vendors of each platforms.
+Please refer to those platforms for information about the privacy policies.
+Depending on the platform, Mixxx Core Developers have access to these reports, which usually contain:
+
+- Operating System version
+- CPU and GPU
+- Symbolic StackTrace of the process
+- List of DLLs loaded by the process
+
+None of these data is stored on Mixxx servers.
+
+## Mixxx update mechanism
+
+The Mixxx application itself doesn't provide an update mechanism.

```

Comment:
We are planning to add an automatic process using Sparkles. 

We may assume this is already in place now to release us from maintenance of this article.  
```suggestion
When downloading or updating the Mixxx application, your IP address and current Mixxx version may be logged temporarily for security and statistical purposes. This information is not used to identify individual users. 
```

Reply:
I think this is redundant, because it's about the main Mixxx Website, not what this section is describing.

Reply:
Once we have Sparkes Mixxx will phone home, accessing a special RSS feed on our website and than starts the actual update if required. 

I like to have this info included buf I don't know

Reply:
Features that are not yet implemented do not belong in a Privacy Policy.